pkg/prometheus/operator.go

```diff
@@ -352,9 +354,19 @@ func (c *Operator) RegisterMetrics(r prometheus.Registerer, reconcileErrorsCount
 		Name: "prometheus_operator_node_address_lookup_errors_total",
 		Help: "Number of times a node IP address could not be determined",
 	})
+	c.nodeEndpointSyncs = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "prometheus_operator_node_syncs_total",
+		Help: "Number of times node endpoints have been synchronized",
```

Comment:
Nit, wdyt of this for the help text: "Number of node endpoints synchronisation."

Reply:
On second thought can't we just have number of successes instead of total? What is the advantage of having total number, versus the failure/success metrics.

Reply:
The advantage is that you can compute the rate of failures more easily: `rate(foo_failed_total[5m])/rate(foo_total[5m])`
vs.
`rate(foo_failed_total[5m])/(rate(foo_failed_total[5m]) + rate(foo_success_total[5m])`

Julius explains it [here](https://youtu.be/_MNYuTNfTb4?t=543).

Reply:
Thanks for the explanation!